### ISSUE TITLE
license text

### ISSUE THREAD

**Issue by ladyada** [adafruit/Adafruit_Windows_Drivers#4]
https://forums.adafruit.com/viewtopic.php?f=19&t=123669

**Comment by dhalbert** [comment]
The text
```
e.	You may redistribute to your 

the Licensed Software only with the modified INF and XML 
files. Your redistribution may be in any form, including 
physical media and Internet downloads.
```
is copied from the SiLabs driver license verbatim. It has the same error.

**Comment by dhalbert** [comment]
Fixed by #6. Found a newer version of SiLabs license without typo (a single missing word: "customer"). Found an official RTF version of GPL3 which looks nicer.
